core/arch/arm/tee/cache.c

```diff
@@ -49,13 +49,15 @@ TEE_Result cache_operation(enum utee_cache_operation op, void *va, size_t len)
 
 	switch (op) {
 	case TEE_CACHEFLUSH:
+#ifdef CFG_PL310
```

Comment:
Yes, I agree. This is indeed confusing. Your proposal about renaming `cache_maintenance_l2()` into `cache_maintenance_outer_l2()` is nice. i will apply it.